coinstacks/bitcoin/api/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@shapeshiftoss/bitcoin-api",
+  "version": "0.1.1",
+  "license": "MIT",
+  "private": true,
+  "main": "dist/bitcoin/api/src/index.js",
+  "source": "src/index.ts",
+  "types": "dist/bitcoin/api/src/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn generate && tsc",
+    "clean": "rm -rf dist node_modules",
+    "generate": "tsoa spec-and-routes"
+  },
+  "dependencies": {
+    "@shapeshiftoss/blockbook": "^0.1.1",
+    "@shapeshiftoss/common-api": "^0.1.1",
+    "@shapeshiftoss/logger": "^0.1.1",
```

Comment:
I already wrote a logger. Where's this one coming from?

Reply:
This is just a really slim logger added locally to this project.